ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/util/ScalablePushUtilTest.java

```diff
@@ -45,47 +44,39 @@
 @RunWith(MockitoJUnitRunner.class)
 public class ScalablePushUtilTest {
 
-  private static final Map<String, Object> overrides = ImmutableMap.of(
-      KsqlConfig.KSQL_QUERY_PUSH_V2_ENABLED, true,
-      KsqlConfig.KSQL_ROWPARTITION_ROWOFFSET_ENABLED, true,
-      "auto.offset.reset", "latest"
-  );
-
   private static final Expression AN_EXPRESSION = mock(Expression.class);
 
   @Mock
   private Query query;
   @Mock
-  private KsqlConfig ksqlConfig;
-  @Mock
   private SingleColumn singleColumn;
   @Mock
   private ColumnReferenceExp columnReferenceExp;
   @Mock
   private KsqlEngine ksqlEngine;
   @Mock
   private Select select;
+  @Mock
+  private KsqlConfig ksqlConfig;
 
   @Before
   public void setUp() {
-    // Otherwise we'd need to mock static ColumnExtractor in each passing test to avoid NPEs
-    when(ksqlConfig.getBoolean(KsqlConfig.KSQL_ROWPARTITION_ROWOFFSET_ENABLED)).thenReturn(false);
+    when(ksqlConfig.getBoolean(KsqlConfig.KSQL_ROWPARTITION_ROWOFFSET_ENABLED)).thenReturn(true);
```

Comment:
This is the default now, right? Can we omit this?

Reply:
For posterity: we communicated and Victoria didn't realize I was using a mocked config- I'll correct behavior of relevant tests to use non-mocked configs or return true as necessary.